common/changes/@microsoft/rush-lib/add-minimum-release-age-support_2025-10-07-10-30.json

```diff
@@ -0,0 +1,10 @@
+{
```

Comment:
Delete this file. All of the Rush packages' changelogs come from changefiles for the `@microsoft/rush` project.